File: ai/src/main/java/org/pingel/causality/examples/midterm.java
package org.pingel.causality.examples;


import org.pingel.bayes.CausalModel;
import org.pingel.bayes.ModelVisualizer;

public class midterm
{
    public static void main(String[] args) {
        
        CausalModel hw4 = new Homework4Model(5, 0.2);

        ModelVisualizer.draw(hw4);
    }
}

File: ai/src/main/scala/org/pingel/causality/examples/midterm.scala
package org.pingel.causality.examples

import org.pingel.causality.CausalModel
import org.pingel.bayes.ModelVisualizer

class midterm {
  
    def main(args: Array[String]) {
        
        val hw4 = new Homework4Model(5, 0.2)

        ModelVisualizer.draw(hw4)
    }
}
